spec/features/posts/post_index.html.erb.spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'rails_helper'
+
+# rubocop:disable Metrics/BlockLength
+RSpec.feature 'Posts index page' do
+  let!(:user) { User.create!(name: 'Alice', bio: 'Maths teacher', photo: 'https://source.unsplash.com/user/c_v_r/1900x800', post_counter: 0) }
+  let!(:user2) { User.create!(name: 'Bob', bio: 'Computer science lecturer', photo: 'https://source.unsplash.com/user/c_v_r/100x100', post_counter: 0) }
+  let!(:posts) do
+    [
+      Post.create(author: user, title: 'Hello there mate1', text: 'This is my first blog post'),
+      Post.create(author: user, title: 'Hello there mate2', text: 'This is my second blog post'),
+      Post.create(author: user, title: 'Hello there mate3', text: 'This is my third blog post'),
+      Post.create(author: user, title: 'Hello there mate4', text: 'This is my fourth blog post'),
+      Post.create(author: user, title: 'Hello there mate5', text: 'This is my fifth  blog post')
+    ]
+  end
+  let!(:comments) do
+    [
+      Comment.create(author: user2, post: posts.first, text: 'Hello Alex this is nice'),
+      Comment.create(author: user2, post: posts.first, text: 'Hi '),
+      Comment.create(author: user2, post: posts.first, text: 'Awesome'),
+      Comment.create(author: user2, post: posts.first, text: 'Fantastic'),
+      Comment.create(author: user2, post: posts.first, text: 'Great'),
+      Comment.create(author: user2, post: posts.first, text: 'Cheers'),
+      Comment.create(author: user2, post: posts.first, text: 'Congrats!')
+    ]
+  end
+  before :each do
+    visit user_posts_path(user)
+  end
+
+  scenario 'Should see post title' do
+    posts.each do |post|
+      expect(page).to have_content(post.title)
+    end
+  end
+
+  scenario 'Should see post body' do
+    expect(page).to have_content(posts.first.text)
+  end
+
+  scenario 'Should see comment count' do
+    expect(page).to have_content("Comments: #{posts.first.comment_counter},")
+  end
+
+  scenario 'Should see likes count' do
+    expect(page).to have_content("Likes:#{posts.first.likes_counter}")
+  end
```

Comment:
- [ ]  According to the project requirements 👇🏻 You should have Tests that assert you can see the number of posts the user has written. Kindly add this 👍🏻 

![Screenshot (1)](https://user-images.githubusercontent.com/79058364/208742598-79005e88-3ac1-4c12-a262-559dea2729bd.png)

Reply:
Hello @BenMukebo, this is covered in [here](https://github.com/kennankole/my-blog-app/blob/development/spec/features/posts/post_index.html.erb.spec.rb#:~:text=expect(page).to%20have_content(%22Number%20of%20posts%3A%20%23%7Buser.posts.count%7D%22))